targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs

```diff
@@ -76,16 +64,20 @@ nuget {
         [Durango,v141,debug] {
             lib: {
                 build\Xbox\Durango\Debug\XPlatXbox.lib;
-                build\Xbox\Durango\Debug\XPlatXbox.pdb;
             }
         };
 
         [Durango,v141,profile] {
             lib: {
                 build\Xbox\Durango\Profile\XPlatXbox.lib;
-                build\Xbox\Durango\Profile\XPlatXbox.pri;
             }
         };
+        
```

Comment:
Thanks for fixing this.
[Question] : Where do the pdb and pri files go? Are paths added to these files or would a user have to add the symbols path manually?